frontend/src/metabase/models/containers/ModelDetailPage/ModelDetailPage.unit.spec.tsx

```diff
@@ -96,31 +140,69 @@ async function setup({ model, collections = [], usedBy = [] }: SetupOpts) {
     );
 
   setupCardsEndpoints(scope, [card]);
-
+  setupActionsEndpoints(scope, model.id(), actions);
   setupCollectionsEndpoints(scope, collections);
 
+  const settings = mockSettings({
+    "experimental-enable-actions": hasActionsEnabled,
+  });
+
+  const storeInitialState: Partial<State> = { settings };
+
+  if (database) {
+    storeInitialState.entities = {
+      databases: {
```

Comment:
Can we make the component load the database by id (`Database.load`) instead of relying on it being in the store?